apps/api-wallet/src/app/auth/auth.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Post, UseGuards, Request, Body } from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { LoginAuthDto } from '../../dtos/LoginUser.dto';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @UseGuards(AuthGuard('local'))
```

Comment:
why is this local?

Reply:
it was my first authguard implementation before changing to authguard jwt

Reply:
my authguard jwt is on wallet controller

Reply:
if this is not being used, remove this lang

Reply:
done